lib/Frontend/CompilerInvocation.cpp

```diff
@@ -358,6 +358,9 @@ static bool ParseLangArgs(LangOptions &Opts, ArgList &Args,
   if (Args.hasArg(OPT_experimental_dependency_include_intrafile))
     Opts.ExperimentalDependenciesIncludeIntrafileOnes = true;
 
+  // TODO: Ignore if enable-experimental-differentiable-programming is false
```

Comment:
From what I've seen in the code {{-enable-experimental-differentiable-programming}} has been added as frontend option but is not really taken into account to enable/disable AutoDiff capabilities. As soon as the opt is taken into account, we can update this to disable {{EnableExperimentalForwardModeDifferentation}} accordingly.